### PR TITLE
enhance: Support old s3 sign method

### PR DESCRIPTION
**Pull request by tosone**

### CODE REVIEW

**Comment by tosone** [comment]
@bigsheeper Could you please review this PR? Thank you.

File: pkg/util/paramtable/service_param.go [hunk 1]
```diff
@@ -1101,6 +1101,7 @@ type MinioConfig struct {
 	Port               ParamItem `refreshable:"false"`
 	AccessKeyID        ParamItem `refreshable:"false"`
 	SecretAccessKey    ParamItem `refreshable:"false"`
+	SignatureVersion   ParamItem `refreshable:"false"`
```
Comment (weiliu1031) [inline pkg/util/paramtable/service_param.go:1104]: Thank you for contributing to the code! Please refer to the other configuration items and add the initialization logic for the configuration items. If it's convenient for you to add some unit tests, you'll find that the missing logic here will cause the configuration to fail to take effect.

**Comment by tosone** [comment]
rerun go-sdk

File: pkg/util/paramtable/service_param.go (hunk 1 above)
Comment (tosone) [reply pkg/util/paramtable/service_param.go:1104]: init logic added in new patch. It's hard to write ut for this pr, <https://min.io/docs/kes/tutorials/configuration/> here, i cannot find minio support old sign method config.

**Pull request closed without merging**